airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_persons.py

```diff
@@ -558,7 +575,7 @@ def test_incremental_rate_limit_max_attempts_exceeded(self, http_mocker: HttpMoc
             state=state,
         )
 
-        assert len(actual_messages.errors) == 1
+        assert len(actual_messages.errors) == 2
```

Comment:
I checked this test and the second error is the aggregate error of all failing streams in the sync. So it would be expected to see the first error trace message followed by the second aggregate error trace which only now happens because we're using a CDK that supports per-stream errors